src/components/popover-container/popover-container-content.style.js

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from 'styled-components';
+import { baseTheme } from '../../style/themes';
+
+const PopoverContainerContentStyle = styled.div`
```

Comment:
Not a blocking comment but just a thought:
Not sure if `PopoverContainerContentStyle` name is proper because it's not really a style, it's a component with applied styles. Thats why I think that widely used convention like `StyledPopoverContainerContent` is better